<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Function reference â€¢ scalop</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-index">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scalop</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/recreate_gbm_metaprograms.html">Recreating the GBM meta-programs</a>
    </li>
  </ul></li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/jlaffy/scalop/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index"><colgroup><col class="alias"><col class="title"></colgroup><tbody><tr><th colspan="2">
          <h2 id="all-functions">All functions <a href="#all-functions" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="GeneInfo.html">GeneInfo()</a></code> </p>
        </td>
        <td><p>Get Genomic Information</p></td>
      </tr><tr><td>
          <p><code><a href="MGH136.html">MGH136</a></code> </p>
        </td>
        <td><p>Glioblastoma patient MGH136</p></td>
      </tr><tr><td>
          <p><code><a href="Markers_Normal.html">Markers_Normal</a></code> </p>
        </td>
        <td><p>Marker gene sets of Normal Cell Types and Pan-Cancer Programs</p></td>
      </tr><tr><td>
          <p><code><a href="NormalBrainSignatures.html">NormalBrainSignatures</a></code> </p>
        </td>
        <td><p>98 Signatures for Normal Brain Cell Types</p></td>
      </tr><tr><td>
          <p><code><a href="NormalBrainSignatures2.html">NormalBrainSignatures2</a></code> </p>
        </td>
        <td><p>98 Signatures for Normal Brain Cell Types</p></td>
      </tr><tr><td>
          <p><code><a href="NormalBrainSignatures3.html">NormalBrainSignatures3</a></code> </p>
        </td>
        <td><p>178 Signatures for Normal Brain Cell Types</p></td>
      </tr><tr><td>
          <p><code><a href="NormalSignatures.html">NormalSignatures</a></code> </p>
        </td>
        <td><p>205 Signatures for Normal Cell Types</p></td>
      </tr><tr><td>
          <p><code><a href="Sapply.html">Sapply()</a></code> </p>
        </td>
        <td><p>sapply wrapper</p></td>
      </tr><tr><td>
          <p><code><a href="Unlist.html">Unlist()</a></code> </p>
        </td>
        <td><p>Unlist, keeping original list or element names</p></td>
      </tr><tr><td>
          <p><code><a href="add_to_dataframe.html">add_to_dataframe()</a></code> </p>
        </td>
        <td><p>Join a dataframe and a named vector</p></td>
      </tr><tr><td>
          <p><code><a href="aggr_gene_expr.html">aggr_gene_expr()</a></code> </p>
        </td>
        <td><p>log-Average expression of genes across all cells</p></td>
      </tr><tr><td>
          <p><code><a href="alias2ensembl.html">alias2ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="alias2entrez.html">alias2entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="alias2refseq.html">alias2refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="alias2symbol.html">alias2symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="alias2ucsc.html">alias2ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="allowedExons.html">allowedExons()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="annomap.html">annomap()</a></code> </p>
        </td>
        <td><p>One-bar annotation ggplot</p></td>
      </tr><tr><td>
          <p><code><a href="as_bulk.html">as_bulk()</a></code> </p>
        </td>
        <td><p>Convert sc-matrix to simulated bulk</p></td>
      </tr><tr><td>
          <p><code><a href="as_bulk_samples.html">as_bulk_samples()</a></code> </p>
        </td>
        <td><p>Convert sc-matrix to simulated bulk samples matrix</p></td>
      </tr><tr><td>
          <p><code><a href="as_four_state_gbm.html">as_four_state_gbm()</a></code> </p>
        </td>
        <td><p>Convert 6-state to 4-state GBM scores</p></td>
      </tr><tr><td>
          <p><code><a href="assign_bulk_states.html">assign_bulk_states()</a></code> </p>
        </td>
        <td><p>Assign Samples to States by High- and Low -frequency of their Cells in each State</p></td>
      </tr><tr><td>
          <p><code><a href="baseScores.html">baseScores()</a></code> </p>
        </td>
        <td><p>Basic Scoring of Matrix by Gene sigs</p></td>
      </tr><tr><td>
          <p><code><a href="bin.html">bin()</a></code> </p>
        </td>
        <td><p>Generate Equally-Sized Bins</p></td>
      </tr><tr><td>
          <p><code><a href="binmatch.html">binmatch()</a></code> </p>
        </td>
        <td><p>Control Signatures From Expression Bins</p></td>
      </tr><tr><td>
          <p><code><a href="bootstrap_summary.html">bootstrap_summary()</a></code> </p>
        </td>
        <td><p>Summary Statistics for Bootstrapped State Fractions</p></td>
      </tr><tr><td>
          <p><code><a href="bootstrapped_state_fractions.html">bootstrapped_state_fractions()</a></code> </p>
        </td>
        <td><p>Bootstrapped State Fractions</p></td>
      </tr><tr><td>
          <p><code><a href="cluster.methods.html">cluster.methods</a></code> </p>
        </td>
        <td><p>agglomeration method in hierarchical clustering
agglomeration method in hierarchical clustering when grouping members into a tree structure.</p></td>
      </tr><tr><td>
          <p><code><a href="col_subset.html">col_subset()</a></code> </p>
        </td>
        <td><p>Keep columns whose names match a pattern</p></td>
      </tr><tr><td>
          <p><code><a href="colcenter.html">colcenter()</a></code> </p>
        </td>
        <td><p>Center a matrix column-wise</p></td>
      </tr><tr><td>
          <p><code><a href="coldetected.html">coldetected()</a></code> </p>
        </td>
        <td><p>Number of non-zero values per column</p></td>
      </tr><tr><td>
          <p><code><a href="combine_group2_dea.html">combine_group2_dea()</a></code> </p>
        </td>
        <td><p>Aggregate `scalop::dea` output for one group against many in turn</p></td>
      </tr><tr><td>
          <p><code><a href="comply.html">comply()</a></code> </p>
        </td>
        <td><p>Apply a Function to All Pairs of Elements in 'x' or Between 'x' and 'y'</p></td>
      </tr><tr><td>
          <p><code><a href="cor.methods.html">cor.methods</a></code> </p>
        </td>
        <td><p>Correlation method
method for computing correlation coefficients</p></td>
      </tr><tr><td>
          <p><code><a href="cpm.html">cpm()</a></code> </p>
        </td>
        <td><p>Convert Counts to Counts-Per-Million (CPM)</p></td>
      </tr><tr><td>
          <p><code><a href="dea.html">dea()</a></code> </p>
        </td>
        <td><p>Differential Expression Analysis</p></td>
      </tr><tr><td>
          <p><code><a href="dims.html">dims()</a></code> </p>
        </td>
        <td><p>&lt;dim&gt; for many matrices</p></td>
      </tr><tr><td>
          <p><code><a href="discrete_colours.html">discrete_colours</a></code> </p>
        </td>
        <td><p>The 297 discrete colours available across different RColorBrewer palettes.</p></td>
      </tr><tr><td>
          <p><code><a href="dist.methods.html">dist.methods</a></code> </p>
        </td>
        <td><p>distance metric
method for computing distances from a matrix object</p></td>
      </tr><tr><td>
          <p><code><a href="doubletCells.html">doubletCells()</a></code> </p>
        </td>
        <td><p>Identify potential doublet cells</p></td>
      </tr><tr><td>
          <p><code><a href="enricher.html">enricher()</a></code> <code><a href="enricher.html">enricher()</a></code> </p>
        </td>
        <td><p>Functional enrichments</p></td>
      </tr><tr><td>
          <p><code><a href="ensembl2symbol.html">`=`()</a></code> </p>
        </td>
        <td><p>ensembl2symbol</p></td>
      </tr><tr><td>
          <p><code><a href="entrez2alias.html">entrez2alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="entrez2ensembl.html">entrez2ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="entrez2refseq.html">entrez2refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="entrez2symbol.html">entrez2symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="entrez2ucsc.html">entrez2ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="expr_housekeeping.html">expr_housekeeping()</a></code> </p>
        </td>
        <td><p>Expression level of Housekeeping Genes</p></td>
      </tr><tr><td>
          <p><code><a href="filter_signatures.html">filter_signatures()</a></code> </p>
        </td>
        <td><p>Filter Out Lowly-Correlated Genes from Signature(s)</p></td>
      </tr><tr><td>
          <p><code><a href="filter_sigs.html">filter_sigs()</a></code> </p>
        </td>
        <td><p>Filter genes in sigs according to reference</p></td>
      </tr><tr><td>
          <p><code><a href="fitBimodal.html">fitBimodal()</a></code> </p>
        </td>
        <td><p>Fit a Bimodal Gaussian Distribution</p></td>
      </tr><tr><td>
          <p><code><a href="flip.html">flip()</a></code> </p>
        </td>
        <td><p>Flip the (nested) elements of a character vector (or list) with its names</p></td>
      </tr><tr><td>
          <p><code><a href="foldchange.html">foldchange()</a></code> </p>
        </td>
        <td><p>fold changes</p></td>
      </tr><tr><td>
          <p><code><a href="frac_mito.html">frac_mito()</a></code> </p>
        </td>
        <td><p>Fraction of a Cell's Expression Level Coming From Mitochondrial Genes</p></td>
      </tr><tr><td>
          <p><code><a href="futurama.html">futurama</a></code> </p>
        </td>
        <td><p>futurama, 12 Discrete Colours</p></td>
      </tr><tr><td>
          <p><code><a href="geneLength.html">geneLength()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="genericModel.html">genericModel()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="getGRCh.html">getGRCh()</a></code> </p>
        </td>
        <td><p>Get hg19 or hg38 information from biomaRt</p></td>
      </tr><tr><td>
          <p><code><a href="get_ensembl.html">get_ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_entrez.html">get_entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_genes.html">get_genes()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_refseq.html">get_refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_signatures.html">get_signatures()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_snn_graph_clusters.html">get_snn_graph_clusters()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_symbols.html">get_symbols()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="get_ucsc.html">get_ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ggDensity.html">ggDensity()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ggmap.html">ggmap()</a></code> </p>
        </td>
        <td><p>gmap wrapper</p></td>
      </tr><tr><td>
          <p><code><a href="ggoutliers.html">ggoutliers()</a></code> </p>
        </td>
        <td><p>Plot model and Outliers</p></td>
      </tr><tr><td>
          <p><code><a href="ggpubr.html">ggpubr</a></code> </p>
        </td>
        <td><p>ggpubr, 3 Discrete Colours</p></td>
      </tr><tr><td>
          <p><code><a href="ggvolcano.html">ggvolcano()</a></code> </p>
        </td>
        <td><p>Volcano plot</p></td>
      </tr><tr><td>
          <p><code><a href="gmap.html">gmap()</a></code> </p>
        </td>
        <td><p>Plot A Heatmap with ggplot2</p></td>
      </tr><tr><td>
          <p><code><a href="graster.html">graster()</a></code> </p>
        </td>
        <td><p>Plot A Heatmap with ggplot2</p></td>
      </tr><tr><td>
          <p><code><a href="grombine.html">grombine()</a></code> </p>
        </td>
        <td><p>Combine ggplot grobs or plots</p></td>
      </tr><tr><td>
          <p><code><a href="grouped_reorder.html">grouped_reorder()</a></code> </p>
        </td>
        <td><p>Grouped Reorder and Ungroup</p></td>
      </tr><tr><td>
          <p><code><a href="grouped_rowcenter.html">grouped_rowcenter()</a></code> </p>
        </td>
        <td><p>Center matrix rows by subsets of columns</p></td>
      </tr><tr><td>
          <p><code><a href="hca.html">hca()</a></code> <code><a href="hca.html">hca_cor()</a></code> <code><a href="hca.html">hca_dist()</a></code> <code><a href="hca.html">hca_tree()</a></code> <code><a href="hca.html">hca_order()</a></code> <code><a href="hca.html">hca_groups()</a></code> <code><a href="hca.html">hca_reorder()</a></code> </p>
        </td>
        <td><p>Hierarchical clustering analysis</p></td>
      </tr><tr><td>
          <p><code><a href="headl.html">headl()</a></code> </p>
        </td>
        <td><p>Left-hand head a matrix or dataframe</p></td>
      </tr><tr><td>
          <p><code><a href="headr.html">headr()</a></code> </p>
        </td>
        <td><p>Right-hand head a matrix or dataframe</p></td>
      </tr><tr><td>
          <p><code><a href="hierarchy.html">hierarchy()</a></code> </p>
        </td>
        <td><p>Cell state hierarchy coordinates</p></td>
      </tr><tr><td>
          <p><code><a href="hms_span.html">hms_span()</a></code> </p>
        </td>
        <td><p>Subtract POSIXt Class Objects in HH:MM:SS Format</p></td>
      </tr><tr><td>
          <p><code><a href="is_accnum.html">is_accnum()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_alias.html">is_alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_annotation.html">is_annotation()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_ensembl.html">is_ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_entrez.html">is_entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_refseq.html">is_refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_symbol.html">is_symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="is_ucsc.html">is_ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="jacFilt.html">jacFilt()</a></code> </p>
        </td>
        <td><p>Filter List By Jaccard</p></td>
      </tr><tr><td>
          <p><code><a href="jaccard.html">Jaccard()</a></code> </p>
        </td>
        <td><p>Compute Jaccard Similarities between Pairs of Character Vectors</p></td>
      </tr><tr><td>
          <p><code><a href="jacmap.html">jacmap()</a></code> </p>
        </td>
        <td><p>Convert a list to a jaccard matrix and plot with `scalop::gmap`</p></td>
      </tr><tr><td>
          <p><code><a href="ldcast.html">ldcast()</a></code> </p>
        </td>
        <td><p>Turn a list into a dataframe</p></td>
      </tr><tr><td>
          <p><code><a href="logRowMeans.html">logRowMeans()</a></code> </p>
        </td>
        <td><p>Row Means of Log2 data</p></td>
      </tr><tr><td>
          <p><code><a href="logcpm.html">logcpm()</a></code> </p>
        </td>
        <td><p>Convert CPM to logCPM</p></td>
      </tr><tr><td>
          <p><code><a href="map_alias.html">map_alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="map_ensembl.html">map_ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="map_entrez.html">map_entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="map_genes.html">map_genes()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="map_refseq.html">map_refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="map_symbol.html">map_symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="markerScores.html">markerScores()</a></code> </p>
        </td>
        <td><p>Score a Matrix with Marker Gene Sets Of Normal Cell Types</p></td>
      </tr><tr><td>
          <p><code><a href="mat01.html">mat01()</a></code> </p>
        </td>
        <td><p>Character List to Binary (membership) Matrix</p></td>
      </tr><tr><td>
          <p><code><a href="maxcol_strict.html">maxcol_strict()</a></code> </p>
        </td>
        <td><p>Strict Column Assign</p></td>
      </tr><tr><td>
          <p><code><a href="metaprograms.html">metaprograms()</a></code> </p>
        </td>
        <td><p>Compute Metaprograms</p></td>
      </tr><tr><td>
          <p><code><a href="msigdb.html">msigdb()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="nSigGenes.html">nSigGenes()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ncols.html">ncols()</a></code> </p>
        </td>
        <td><p>&lt;ncol&gt; for many matrices</p></td>
      </tr><tr><td>
          <p><code><a href="nrows.html">nrows()</a></code> </p>
        </td>
        <td><p>&lt;nrow&gt; for many matrices</p></td>
      </tr><tr><td>
          <p><code><a href="ntop.html">ntop()</a></code> </p>
        </td>
        <td><p>Select top n elements</p></td>
      </tr><tr><td>
          <p><code><a href="ohmySNNclusters.html">ohmySNNclusters()</a></code> </p>
        </td>
        <td><p>SNN Graph-Based Clustering</p></td>
      </tr><tr><td>
          <p><code><a href="ohmyqc.html">ohmyqc()</a></code> </p>
        </td>
        <td><p>Quality control statistics for single cell RNA seq data</p></td>
      </tr><tr><td>
          <p><code><a href="ohmytsne.html">ohmytsne()</a></code> </p>
        </td>
        <td><p>A plot-friendly output for the Barnes-Hut implementation of t-SNE.</p></td>
      </tr><tr><td>
          <p><code><a href="outliers.html">outliers()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="overlap.html">Overlap()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="pairup.html">pairup()</a></code> </p>
        </td>
        <td><p>List combinations</p></td>
      </tr><tr><td>
          <p><code><a href="pal19.html">pal19</a></code> </p>
        </td>
        <td><p>pal19, 19 Pretty Discrete Colours</p></td>
      </tr><tr><td>
          <p><code><a href="pal28.html">pal28</a></code> </p>
        </td>
        <td><p>pal28, 28 Pretty Discrete Colours</p></td>
      </tr><tr><td>
          <p><code><a href="palG34R.html">palG34R</a></code> </p>
        </td>
        <td><p>palG34R, a palette of reds and blues for cases within two groups</p></td>
      </tr><tr><td>
          <p><code><a href="pca.html">pca()</a></code> </p>
        </td>
        <td><p>Principal Component Analysis</p></td>
      </tr><tr><td>
          <p><code><a href="plot_hierarchy.html">plot_hierarchy()</a></code> </p>
        </td>
        <td><p>Plot State Hierarchy (Quadrant plot)</p></td>
      </tr><tr><td>
          <p><code><a href="programs.html">programs()</a></code> </p>
        </td>
        <td><p>Find cell clusters and retrieve significant expresion programs</p></td>
      </tr><tr><td>
          <p><code><a href="range_rowmeans.html">range_rowmeans()</a></code> </p>
        </td>
        <td><p>Range of the rowMeans of a matrix</p></td>
      </tr><tr><td>
          <p><code><a href="refseq2alias.html">refseq2alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="refseq2ensembl.html">refseq2ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="refseq2entrez.html">refseq2entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="refseq2symbol.html">refseq2symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="refseq2ucsc.html">refseq2ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="row_subset.html">row_subset()</a></code> </p>
        </td>
        <td><p>Keep rows whose names match a pattern</p></td>
      </tr><tr><td>
          <p><code><a href="rowcenter.html">rowcenter()</a></code> </p>
        </td>
        <td><p>Center a matrix row-wise</p></td>
      </tr><tr><td>
          <p><code><a href="rowdetected.html">rowdetected()</a></code> </p>
        </td>
        <td><p>Number of non-zero values per row</p></td>
      </tr><tr><td>
          <p><code><a href="sample_strict.html">sample_strict()</a></code> </p>
        </td>
        <td><p>Assign Rows and Sample</p></td>
      </tr><tr><td>
          <p><code><a href="scdata.html">scdata</a></code> </p>
        </td>
        <td><p>scRNA-seq data from a patient with Glioblastoma</p></td>
      </tr><tr><td>
          <p><code><a href="score.html">score()</a></code> </p>
        </td>
        <td><p>Score matrix columns for groups of rows</p></td>
      </tr><tr><td>
          <p><code><a href="setColNames.html">setColNames()</a></code> </p>
        </td>
        <td><p>Set the colnames in an object</p></td>
      </tr><tr><td>
          <p><code><a href="setRowNames.html">setRowNames()</a></code> </p>
        </td>
        <td><p>Set the rownames in an object</p></td>
      </tr><tr><td>
          <p><code><a href="sigScores.html">sigScores()</a></code> </p>
        </td>
        <td><p>Score a Matrix by Gene sigs (Signatures)</p></td>
      </tr><tr><td>
          <p><code><a href="simpsons.html">simpsons</a></code> </p>
        </td>
        <td><p>simpsons, 16 Discrete Colours</p></td>
      </tr><tr><td>
          <p><code><a href="state_fractions.html">state_fractions()</a></code> </p>
        </td>
        <td><p>State Fractions (by group)</p></td>
      </tr><tr><td>
          <p><code><a href="state_high_genes.html">state_high_genes()</a></code> </p>
        </td>
        <td><p>Genes Associated with High Frequency of a Cellular State</p></td>
      </tr><tr><td>
          <p><code><a href="subsplit.html">subsplit()</a></code> </p>
        </td>
        <td><p>Split a vector, list, matrix or dataframe into  into a list of matrices or nested list according to substring matches.</p></td>
      </tr><tr><td>
          <p><code><a href="substri.html">substri()</a></code> </p>
        </td>
        <td><p>Extract substrings between delimiters by position</p></td>
      </tr><tr><td>
          <p><code><a href="symbol2alias.html">symbol2alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="symbol2ensembl.html">symbol2ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="symbol2entrez.html">symbol2entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="symbol2refseq.html">symbol2refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="symbol2ucsc.html">symbol2ucsc()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="taill.html">taill()</a></code> </p>
        </td>
        <td><p>Left-hand tail a matrix or dataframe</p></td>
      </tr><tr><td>
          <p><code><a href="tailr.html">tailr()</a></code> </p>
        </td>
        <td><p>Right-hand tail a matrix or dataframe</p></td>
      </tr><tr><td>
          <p><code><a href="theme_scalop.html">theme_scalop()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="totalExons.html">totalExons()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="tpm.html">tpm()</a></code> </p>
        </td>
        <td><p>Convert logTPM to TPM</p></td>
      </tr><tr><td>
          <p><code><a href="ucsc2alias.html">ucsc2alias()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ucsc2ensembl.html">ucsc2ensembl()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ucsc2entrez.html">ucsc2entrez()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ucsc2refseq.html">ucsc2refseq()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="ucsc2symbol.html">ucsc2symbol()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="unique_sample_names.html">unique_sample_names()</a></code> </p>
        </td>
        <td><p>Get unique sample names</p></td>
      </tr><tr><td>
          <p><code><a href="unlogcpm.html">unlogcpm()</a></code> </p>
        </td>
        <td><p>Convert logCPM to CPM</p></td>
      </tr><tr><td>
          <p><code><a href="unlogtpm.html">logtpm()</a></code> <code><a href="unlogtpm.html">unlogtpm()</a></code> </p>
        </td>
        <td><p>Convert TPM to logTPM</p></td>
      </tr><tr><td>
          <p><code><a href="vannomap.html">vannomap()</a></code> </p>
        </td>
        <td><p>One-bar annotation ggplot</p></td>
      </tr><tr><td>
          <p><code><a href="which_annotation.html">which_annotation()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr><tr><td>
          <p><code><a href="wilcoxtest.html">wilcoxtest()</a></code> </p>
        </td>
        <td><p>wilcoxon-tests</p></td>
      </tr><tr><td>
          <p><code><a href="x2y.html">x2y()</a></code> </p>
        </td>
        <td><p>FUNCTION_TITLE</p></td>
      </tr></tbody></table></div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Julie Laffy.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer></div>

  


  

  </body></html>

